shap/plots/_waterfall.py

```diff
@@ -9,11 +12,29 @@
 from ._labels import labels
 
```

Comment:
I would suggest adding a Type Alias for objects that can represent a colour, so that the type hint does not need to be repeated.

For example:
```suggestion
ColorSpec = Union[np.ndarray, str, List[float]]
```